engine/class_modules/sc_death_knight.cpp

```diff
@@ -6115,7 +6060,7 @@ struct scourge_strike_t : public scourge_strike_base_t
     scourge_strike_shadow( new scourge_strike_shadow_t( p ) )
   {
     parse_options( options_str );
-
+    base_multiplier *= 1.0 + p -> spec.scourge_strike_2 -> effectN( 1 ).percent();
```

Comment:
I hoped applying the multiplier to the base alone would be enough but it wasn't applying to the shadow component so I had to add the multiplier to each component.

Reply:
Don't forget clawing shadows, it's also in the list of spells affected by rank 2.